### PR TITLE
update ROCm naming

### PR DESCRIPTION
**Pull request by LisaDelaney**

### CODE REVIEW

File: docs/.doxygen/Doxyfile [hunk 1]
```diff
@@ -44,7 +44,7 @@ PROJECT_NUMBER         = v4.0.5.0
 # for a project that appears at the top of each page and should give viewer a
 # quick idea about the purpose of the project. Keep the description short.
 
-PROJECT_BRIEF          = "prototype interfaces compatible with ROCm platform and HiP"
+PROJECT_BRIEF          = "prototype interfaces compatible with ROCm software and HiP"
```
Comment (daineAMD) [inline docs/.doxygen/Doxyfile]: I think "HiP" should be "HIP", might as well update here.

**Review by TorreZuk — APPROVED** [review]
Fine by me

**Review by daineAMD — APPROVED** [review]

File: docs/.doxygen/Doxyfile (hunk 1 above)
Comment (LisaDelaney) [reply docs/.doxygen/Doxyfile]: agreed!

Comment (daineAMD) [reply docs/.doxygen/Doxyfile]: Thanks, looks good. Feel free to squash and merge.

**Pull request merged**